src/ElasticEngine.php

```diff
@@ -348,7 +348,7 @@ public function map(Builder $builder, $results, $model)
      */
     public function getTotalCount($results)
     {
-        return $results['hits']['total']['value'];
+        return $results['hits']['total']['value'] ?? 0;
```

Comment:
The change is just one line and doesn't affect the existing functionality. Our team is blocked rn so we'd really appreciate it if we could get this change fixed :)